libpod/runtime_ctr.go

```diff
@@ -460,3 +465,27 @@ func (r *Runtime) GetLatestContainer() (*Container, error) {
 	}
 	return ctrs[lastCreatedIndex], nil
 }
+
+// reserveLabels walks the list o fcontainers and reserves the label, so new containers will not
+// get them.
+func (r *Runtime) reserveLabels() error {
```

Comment:
Can you throw a TODO on here to optimize this, performance-wise? I might poke at it over the weekend if I get bored

Reply:
Actually, idea - can we do this only once, in `makeRuntime()`? That way, if we do multiple things with the `libpod` runtime, we don't need to repeatedly reserve labels

Reply:
Added.